src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -309,6 +332,10 @@ export class ServiceScene extends SceneObjectBase<ServiceSceneState> {
     const breakdownView = getDrilldownSlug();
     const breakdownViewDef = breakdownViewsDefinitions.find((v) => v.value === breakdownView);
 
+    if (!body) {
```

Comment:
the body is always added first in the activation, and setState is synchronous, so it should never be undefined at runtime. Figured a hard error is best way to sniff out if that changes in the future?